R/correctionsataglance-data.R

```diff
@@ -203,7 +203,7 @@ parseCorrThresholds <- function(reportObject, timezone){
       type <- rep(threshold[['type']], times=nrow(periods))
       startDates <- periods[['startTime']]
       endDates <- periods[['endTime']]
-      value <- periods[['referenceValue']]
+      value <- if(isEmptyOrBlank(periods[['referenceValue']])){NA} else{periods[['referenceValue']]}
```

Comment:
Also, ifelse can work on things  > length 1, can't it? Apparently it returns a vector the same length as what you ask it to check? (although it doesn't work in this case) http://stackoverflow.com/questions/1335830/why-cant-rs-ifelse-statements-return-vectors